openstudiocore/src/openstudio_app/Resources/hvaclibrary/hvac_library.osm

```diff
@@ -1940,6 +1940,368 @@ OS:WaterHeater:Mixed,
+
+OS:Curve:Quadratic,
+  {489134e3-6c2a-4475-8f8b-a19ad0b8c965}, !- Handle
+  HPWH COP Func of Water Flow Frac,                      !- Name
+  1,                                      !- Coefficient1 Constant
+  0,                                      !- Coefficient2 x
+  0,                                      !- Coefficient3 x**2
+  0,                                      !- Minimum Value of x {BasedOnField A2}
+  1;                                      !- Maximum Value of x {BasedOnField A2}
+
+OS:Curve:Quadratic,
+  {248ab195-cfb7-4c64-89bb-2018882ad681}, !- Handle
+  Part Load Frac Correlation,                      !- Name
+  0.75,                                   !- Coefficient1 Constant
+  0.25,                                   !- Coefficient2 x
+  0,                                      !- Coefficient3 x**2
+  0,                                      !- Minimum Value of x {BasedOnField A2}
+  1;                                      !- Maximum Value of x {BasedOnField A2}
+
+OS:WaterHeater:Mixed,
+  {5a6a31ef-f662-4452-925d-855d251337c0}, !- Handle
+  HPWH Tank,                   !- Name
+  0.3785,                                 !- Tank Volume {m3}
+  {82f28594-c0f3-4288-ac71-545c739367a8}, !- Setpoint Temperature Schedule Name
+  2,                                      !- Deadband Temperature Difference {deltaC}
+  82.22,                                  !- Maximum Temperature Limit {C}
+  Cycle,                                  !- Heater Control Type
+  845000,                                 !- Heater Maximum Capacity {W}
+  ,                                       !- Heater Minimum Capacity {W}
+  ,                                       !- Heater Ignition Minimum Flow Rate {m3/s}
+  ,                                       !- Heater Ignition Delay {s}
+  NaturalGas,                             !- Heater Fuel Type
+  0.8,                                    !- Heater Thermal Efficiency
+  ,                                       !- Part Load Factor Curve Name
+  20,                                     !- Off Cycle Parasitic Fuel Consumption Rate {W}
+  NaturalGas,                             !- Off Cycle Parasitic Fuel Type
+  0.8,                                    !- Off Cycle Parasitic Heat Fraction to Tank
+  ,                                       !- On Cycle Parasitic Fuel Consumption Rate {W}
+  NaturalGas,                             !- On Cycle Parasitic Fuel Type
+  ,                                       !- On Cycle Parasitic Heat Fraction to Tank
+  Schedule,                               !- Ambient Temperature Indicator
+  {fec1f40a-140f-4541-b3e7-8018896e49ac}, !- Ambient Temperature Schedule Name
+  ,                                       !- Ambient Temperature Thermal Zone Name
+  ,                                       !- Ambient Temperature Outdoor Air Node Name
+  6,                                      !- Off Cycle Loss Coefficient to Ambient Temperature {W/K}
```

Comment:
@kbenne Is there a function to set this?  I couldn't find it in the files below.